frontends/ol-utilities/src/date/utils.ts

```diff
@@ -52,36 +52,11 @@ const formatDurationClockTime = (value: string | moment.Duration) => {
   return values.join(":")
 }
 
-type TimeUntil = {
-  ms: number
-  /**
-   * The number of days until the date.
-   * This is a float! It can be fractional.
-   */
-  days: number
-  isToday: boolean
-  isTomorrow: boolean
-}
-const getTimeUntil = (date: string): TimeUntil | null => {
+const calendarDaysUntil = (date: string): number | null => {
   const x = moment(date)
   if (!x.isValid()) return null
-  const today = {
-    start: moment().startOf("day"),
-    end: moment().endOf("day"),
-  }
-  const tomorrow = {
-    start: today.end,
-    end: moment().add(1, "days").endOf("day"),
-  }
-  const diff = x.diff(moment()) // ms
-
-  const duration = moment.duration(diff)
-  return {
-    ms: diff,
-    days: duration.asDays(),
```

Comment:
This was wrong: we want calendar days. Duration as a days doesn't take timezones into account. We want the difference dates on a calendar, expressed as days. 

`calendarDaysUntil` does that.